docs/colyseus/demo/babylonjs-editor/hide-and-seek.md

```diff
@@ -0,0 +1,83 @@
+# Hide and Seek Tech Demo
+
+!!! requirement "Requirements"
+    - Node.js v14.0 or higher
+    - Colyseus 0.14.20
+    - Babylon.js Editor 4.4.0 or higher
+
+The purpose of this technical demo is to show how to use Colyseus with the Babylon.js Editor. Following documentation covers installing and using the Colyseus client SDK as well as locally running a Colyseus server. The demo is designed to work with Colyseus version 0.14.20 and [Babylon.js Editor version 4.4.0](http://editor.babylonjs.com/).
+
+**[Download demo](https://github.com/colyseus/babylonjs-hide-and-seek/archive/master.zip)** ([View source code](https://github.com/colyseus/babylonjs-hide-and-seek/))
+
+![Title](hide-and-seek/title.png)
+
+## Getting Started
+
+### Installing & Launching a local server
+You need to install and launch the server from the **provided Server directory** for this demo to function properly. Simply follow the instructions found underneath [“Running the server locally” in the Unity3d section of these docs](/getting-started/unity3d-client/#running-the-server-locally). 
+
+!!! note "Note"
+    - The server needs to be installed first before attempting to run or open the project in the editor otherwise errors will result during the compilation process. This is due to client code referencing class types from the server code for development convenience.
+    - When opening the demo in the Babylon.js Editor some `.js` files will be generated in the **Server** folder alongside their original `.ts` source files. When launching the server locally (via `npm start`) those `.js` files are removed to prevent runtime issues.
+
+### Colyseus Server Settings
+All server settings are defined in `.env` files located in the Babylon.js Editor workspace here:
+
+![Server Settings](hide-and-seek/server-settings.png)
```

Comment:
Need to rename the files from `.PNG` to `.png` for them to display correctly on the final build